Automattic/sensei
# Fix JavaScript error for a quiz containing a zero-graded question

Fixes #2373.

## Testing
1. Create a question where the _Question grade_ is 0.
2. Add the question to a lesson's quiz.
3. Set the quiz to not grade automatically.
4. On the front-end, take the quiz.
5. In _Sensei_ > _Grading_, find the quiz and click the _Grade Quiz_ button.
6. Ensure there is no JS error in the console.
